manifests/init.pp

```diff
@@ -116,6 +120,46 @@
     $rancid_path_env_real = $rancid_path_env
   }
 
+  if $vcs == 'USE_DEFAULTS' {
+    $vcs_real = $default_vcs
+  } else {
+    validate_re($vcs, '^(cvs|svn|git)$', "rancid::vcs is ${vcs} and must be one of [cvs, svn, git]")
+    $vcs_real = $vcs
+  }
+
+  if $vcsroot == 'USE_DEFAULTS' {
+    case $vcs_real {
+      default: {
```

Comment:
You got to have a default, though this is impossible to get to. Would change out the notify for a `fail()`.